.github/actions/prep-release/action.yml

```diff
@@ -24,13 +24,16 @@ inputs:
   silent:
     description: "Set a placeholder in the changelog and don't publish the release."
     required: false
-    type: boolean
```

Comment:
I'm assuming this was an unintended removal?

Reply:
Nah that was on purpose. Types aren't supported for action inputs and the GitHub actions extension in VS Code was complaining about it